### ISSUE TITLE
natnael

### ISSUE THREAD

**Issue by darknodi** [natnael-wubet/pubbet#1]
@natnael-wubet hi. can you make. me. an script. 

in this coordination

thread.ps1 -a 1.1.1.1 -p 0,999 -t 10 

adress port ranger thread 10 

message started 

threading......

**Comment by natnael-wubet** [comment]
Are u looking for a port scanner?

**Comment by darknodi** [comment]
no brother.

a fully script in bash or shell script 

for multi threading ranged servers by a single port. 

like 192.168.1.1-254 80 10 thread.

sh script.sh 192.168.1.1-254 or 192.168.1,254 80 10 thread 

threading started. @natnael-wubet

**Comment by natnael-wubet** [comment]
Yeah but on 1 port u can only listen for client one time but the hundler
for each clients can be in thread, if you are familiar in powershell, read
about start-job article,i hope it ill answer all you need
